selector_server:
  ros__parameters:
    use_sim_time: True
    base_frame_id: "base_footprint"
    odom_frame_id: "odom"
    polygons: ["PolygonStop"]
    CostmapBased:
      points_right: "[[1.5, -1.5], [1.5, 0], [-1.5, 0], [-1.5, -1.5]]"
      points_left: "[[1.5, 0], [1.5, 1.5], [-1.5, -1.5], [-1.5, 0]]"
      max_points: 5
      visualize: True
      polygon_pub_topic: "polygon_en2"
